### PR TITLE
remove invalid_hostname check (fate #319639)

### PR DESCRIPTION
**Pull request by wfeldt**
this basically un-does:
https://github.com/yast/yast-yast2/pull/407

install system has been fixed with these 2 PRs:
https://github.com/openSUSE/linuxrc/pull/108
https://github.com/openSUSE/installation-images/pull/101

### CODE REVIEW

**Comment by mchf** [comment]
LGTM

File: package/yast2.changes [hunk 1]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Mon May  9 13:41:51 CEST 2016 - snwint@suse.de
+
+- remove invalid_hostname check (fate #319639)
```
Comment (mvidner) [inline package/yast2.changes]: I think the guardian robots do not like the space between "fate" and "#".
You can run the check locally with `rake check:changelog`

Comment (wfeldt) [inline package/yast2.changes]: yes, you are right

**Comment by mvidner** [comment]
(Removing) code looks good :)
Please make mergeable, then LGTM.

**Pull request merged**